i18n/en-US.yml

```diff
@@ -28,16 +28,88 @@ _name: English
 components:
   DefaultItinerary:
     clickDetails: Click to view details
-    # Use ordered placeholders for the departure-arrival string
-    # (this will accommodate right-to-left languages by swapping the order in this string).
-    departureArrivalTimes: "{startTime}—{endTime}"
     # Use ordered placeholders when multiple modes are involved
     # (this will accommodate right-to-left languages by swapping the order/separator in this string).
     multiModeSummary: "{accessMode} to {transitMode}"
-    # If trip is less than one hour only display the minutes.
-    tripDurationFormatZeroHours: "{minutes, number} min"
-    # TODO: Distinguish between one hour (singular) and 2 hours or more?
-    tripDurationFormat: "{hours, number} hr {minutes, number} min"
+  ItinerarySummary:
+    fareCost: "{useMaxFare, select, 
+      true {{minTotalFare} - {maxTotalFare}}
+      other {{minTotalFare}}
+    }"
+  NarrativeItinerariesHeader:
+    numIssues: "{issueNum, number} issues"
+    resultText: "{pending, select,
+      true {Finding your options...}
+      other {
+        {itineraryNum, number} {itineraryNum, plural,
+          one {itinerary found}
+          other {itineraries found}
+        }
+      }
+    }"
+    selectArrivalTime: Arrival time
+    selectBest: Best option
+    selectCost: Cost
+    selectDepartureTime: Departure time
+    selectDuration: Duration
+    selectWalkTime: Walk time
+    titleText: "{pending, select, 
+      true {Finding your options...}
+      other {
+        {itineraryNum, number} {itineraryNum, plural,
+          one {itinerary}
+          other {itineraries}}
+        {issueNum, plural,
+          =0 {found}
+          one {(and {issueNum, number} issue) found}
+          other {(and {issueNum,number} issues) found}
+        }
+      }
+    }"
+    viewAll: View all options
+  PlanFirstLastButtons:
+    # Note to translator: these values are width-constrained.
+    first: First
+    last: Last
+    next: Next
+    previous: Previous
+  RealtimeAnnotation:
+    ignoreServiceDelays: Apply service delays
+    delaysNotShownInResults: "Your trip results are currently being affected by service delays.
+      These delays do not factor into travel times shown below."
+    delaysShownInResults: "Your trip results have been adjusted based on real-time
+      information. Under normal conditions, this trip would take <b>{normalDuration}</b>
+      using the following routes: {routes}."
+    ignoreServiceDelays: Ignore service delays
+    serviceUpdate: Service update
+  SaveTripButton:
+    cantSaveText: Cannot save
+    cantSaveTooltip: Only itineraries that include transit and no rentals or ride hailing can be monitored.
+    saveTripText: Save trip
+    signInText: Sign in to save trip
+    signInTooltip: Please sign in to save trip.
+  SimpleRealtimeAnnotation:
+    usingRealtimeInfo: This trip uses real-time traffic and delay information
+  TabbedItineraries:
+    optionNumber: "Option {optionNum,number}"
```

Comment:
```suggestion
    optionNumber: "Option {optionNum, number}"
```

Reply:
addressed in https://github.com/opentripplanner/otp-react-redux/pull/430/commits/ad8ea3326848a5ac077e01898d7f7657412fda84